src/pattern-library/components/patterns/ColorFoundations.js

```diff
@@ -25,39 +21,56 @@ const backgroundExamples = [
   'grey-7',
   'grey-8',
   'grey-9',
```

Comment:
Not related to this PR, but why is 'black' not shown here?

Reply:
We don't actually currently use black in any of our design components, is why :)